src/components/App/App.js

```diff
@@ -18,13 +18,13 @@ const App = () => {
       .then(courses => setCourses(courses))
       .catch(error => setError(error))
     setIsLoading(false)
-  })
+  }, [])
```

Comment:
Do we need the empty array second argument on this line if we aren't implementing any skipping effects?

Reply:
![Screen Shot 2020-01-06 at 9 37 46 PM](https://user-images.githubusercontent.com/47537744/71868826-18292500-30cd-11ea-9b58-91e9f4697cef.png)
So I noticed @ericwm76  had this line before. I know my text editor was giving me this error. The risk of "infinite chain of updates" sounded intense to me, I would love to discuss this further if you do not think we need it. @chrisdbasham317 would you mind elaborating more on "skipping effects" and how it relates to the array? I just want to be sure I fully understand why and when to use this.